pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1588,6 +1591,7 @@ func (c *VolumeBindingChecker) predicate(pod *v1.Pod, meta algorithm.PredicateMe
 		return false, failReasons, nil
 	}
 
+	metrics.VolumeBindingPredicateLatency.Observe(metrics.SinceInMicroseconds(start))
```

Comment:
I think we should also record latency when predicate returns failReasons because it still has to do lots of calculations in FindPodVolumes.